module/behaviour/strategy/PS3Walk/src/Joystick.hpp

```diff
@@ -11,14 +11,14 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <http://www.gnu.org/licenses/>
 
-#ifndef __JOYSTICK_HH__
-    #define JOYSTICK_HH_
+#ifndef __JOYSTICK_HPP__
+#define __JOYSTICK_HPP__
```

Comment:
It's mad at you because of the `__THING__` Things that start with a double underscore are reserved for use by the compiler. That's why clang-tidy removed it in the first place, but it only removed it for the `#define` which made the indentation go all screwy because it was no longer a valid macroguard
```suggestion
#ifndef MODULE_BEHAVIOUR_STRATEGY_PS3WALK_JOYSTICK_HPP
#define MODULE_BEHAVIOUR_STRATEGY_PS3WALK_JOYSTICK_HPP
```

Reply:
Ohhh, that makes a lot of sense. Thanks